contracts/storages/SymbolStorage.sol

```diff
@@ -10,19 +10,54 @@ struct Symbol {
 	bool isValid;
 	uint256 minAcceptableQuoteValue;
 	uint256 minAcceptablePortionLF;
-	uint256 tradingFee;
```

Comment:
This is breaking change. Why rename it? Also defaultFee is equally ambiguous as tradingFee. Maybe even more confusing. Implies there is some other tradingFee that is supposed to apply. Is there?